ui/components/common/CommonToolbar.js

```diff
@@ -50,6 +52,8 @@ class CommonToolbar extends Component {
                         type="text"
                         placeholder="Filter by Name"
                         className={toolbarFilter}
+                        value={get(this.props.filter, "name", "")}
+                        style={{ height: "26px" }}
```

Comment:
This should be declared in `.toolbarFilter` css class, but I think it's not needed anyway.